makefile

```diff
@@ -12,13 +12,21 @@ osx:
 install:
 	go install .
 
+os=$(shell uname)
+ifeq ($(os),Darwin)
+  DEV_BANNO_IP=192.168.99.100
+  LOCAL_BANNO_IP=192.168.99.1
+else
+  DEV_BANNO_IP=127.0.0.1
+  LOCAL_BANNO_IP=127.0.0.1
+endif
+
```

Comment:
This isn't using big, just what big had configured for the DNS container (which we can remove, mentioned in the other comment).